testcases/open_posix_testsuite/conformance/interfaces/fscanf/1-1.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file is licensed under the GPL license. For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *  When there are excess arguments as compared to the directives in the
+ *	control string format, they will be ignored.
+ *
+ * method:
+ *  -open file in write mode
+ *  -write 3 integers into the file
+ *  -close the file
+ *  -open the same file in read mode
+ *  -read only 2 integers from the file using fscanf() while giving 3 pointer
+ *   arguments,
+ *  -third integer value will be ignored
+ *  -check the 2 read integer values
+ */
+
+#define _XOPEN_SOURCE 700
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <limits.h>
+#include "posixtest.h"
+
+#define BUF_SIZE 1024
+#define TNAME "fscanf/1-1.c"
+#define FNAME "/tmp/in_file"
```

Comment:
The files in tmp should have reasonably unique name, most tests use pid to make names unique, it's also good idea to include name of the test in the temporary file name such as "fscanf_1-1_$pid".